mlrun/config.py

```diff
@@ -593,6 +593,22 @@
                 "max_replicas": 1,
             },
         },
+        "controller_stream_args": {
+            "v3io": {
+                "shard_count": 2,
+                "retention_period_hours": 24,
+                "num_workers": 1,
+                "min_replicas": 2,
+                "max_replicas": 2,
+            },
+            "kafka": {
+                "partition_count": 8,
+                "replication_factor": 1,
+                "num_workers": 2,
+                "min_replicas": 1,
+                "max_replicas": 4,
+            },
+        },
```

Comment:
In order to keep the same performance lets try this configuration to be the default. 
Lets talk on this before you changing to this configuration.
```suggestion
        "controller_stream_args": {
            "v3io": {
                "shard_count": 10,
                "retention_period_hours": 24,
                "num_workers": 1,
                "min_replicas": 1,
                "max_replicas": 1,
            },
            "kafka": {
                "partition_count": 10,
                "replication_factor": 1,
                "num_workers": 10,
                "min_replicas": 1,
                "max_replicas": 1,
            },
        },
```